spec/models/meter_spec.rb

```diff
@@ -100,10 +100,10 @@
 
     context 'when finding meters for review' do
       let(:meter_review) { create(:meter_review) }
-      let(:electricity_meter_reviewed) { create(:electricity_meter, dcc_meter: true, meter_review: meter_review, mpan_mprn: 1_234_567_890_111) }
-      let(:electricity_meter_not_reviewed) { create(:electricity_meter, dcc_meter: true, mpan_mprn: 1_234_567_890_222, meter_review_id: nil, consent_granted: false) }
-      let(:electricity_meter_not_dcc) { create(:electricity_meter, dcc_meter: false, mpan_mprn: 1_234_567_890_333) }
-      let(:electricity_meter_consent_granted_already) { create(:electricity_meter, dcc_meter: true, consent_granted: true, mpan_mprn: 1_234_567_890_444) }
+      let(:electricity_meter_reviewed) { create(:electricity_meter, dcc_meter: :smets2, meter_review: meter_review, mpan_mprn: 1_234_567_890_111) }
```

Comment:
Small thing but should there be a test that its finding meters marked as `:other` as we're still only checking the 2 original states. Same possibly true for some of the other specs too?

Reply:
added some other meters